dev-packages/application-manager/src/generator/webpack-generator.ts

```diff
@@ -83,7 +83,16 @@ module.exports = {
             },
             {
                 test: /\\.useable\\.css$/,
-                loader: 'style-loader/useable!css-loader'
+                use: [
+                  {
+                    loader: 'style-loader/useable',
+                    options: {
+                      singleton: true,
+                      attrs: { id: 'theme' },
```

Comment:
maybe a bit more specific name, like `theia-theme`, not sure whether it can collide with styles element installed by some 3rd party extensions